modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1042,4 +980,46 @@ a|The query used a procedure that generated a warning.
 m|WARNING
 |Category
 m|GENERIC
-|===
\ No newline at end of file
+|===
+
+[#_neo_clientnotification_statement_RepeatedRelationshipReference]
+=== Neo.ClientNotification.Statement.RepeatedRelationshipReference
+
+.Notification category details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Statement.RepeatedRelationshipReference
+|Title
+a|The query returns no results because a relationship variable is bound more than once.
+|Severity
+m|WARNING
+|Category
+m|GENERIC
+|===
+
```

Comment:
Maybe we could add an example where we can give a suggestion for improvement, like this maybe:

```suggestion
Query::
+
[source,cypher]
----
MATCH (:A)-[r]->(), ()-[r]->(:B) RETURN r
----
Description of the returned code::
A relationship is referenced more than once in the query, which leads to no results because relationships must not occur more than once in each result. (Relationship `r` was repeated)
Suggestions for improvement::
In order to find all relationships which have a start node with label `A` and end node with label `B`, it could be rewritten to the following:
+
[source, cypher, role="noplay"]
----
MATCH (:A)-[r]->(:B) RETURN r
----
```